zanata-war/src/main/java/org/zanata/service/impl/LocaleServiceImpl.java

```diff
@@ -90,12 +91,17 @@ public LocaleServiceImpl(LocaleDAO localeDAO, ProjectDAO projectDAO,
 
     public static Map<LocaleId, String> getLocaleAliasesByIteration(HProjectIteration iteration) {
         Map<LocaleId, String> localeAliases;
```

Comment:
Hey @aeng I got your point but I was busy with the other ticket so I couldn't answer this. 
By the way, I was about to write this comment to you. 

We looked at this PR with @carlosmunoz First, tried to reproduce this exception but we couldn't and Carlos realized that this NullPointerException is caused by Seam. So he is considering closing the PR.